presidio_evaluator/evaluation/evaluator.py

```diff
@@ -344,3 +347,119 @@ def f_beta(precision: float, recall: float, beta: float) -> float:
         return ((1 + beta ** 2) * precision * recall) / (
             ((beta ** 2) * precision) + recall
         )
+
+    class Plotter:
+        """
+        Plot scores (f2, precision, recall) and errors (false-positivies, false-negatives) 
+        for a PII detection model evaluated via Evaluator
+
+        :param model: Instance of a fitted model (of base type BaseModel)
+        :param results: results given by evaluator.calculate_score(evaluation_results)
+        :param output_folder: folder to store plots and errors in
+        :param model_name: name of the model to be used in the plot title
+        """
+
+        def __init__(self, model, results, output_folder, model_name):
+            self.model = model
+            self.results = results
+            self.output_folder = output_folder
+            self.model_name = model_name.replace("/", "-")
+            self.errors = results.model_errors
+
+        def plot_scores(self) -> None:
+            """
+            Plots per-entity recall, precision, or F2 score for evaluated model. 
+            :param plot_type: which metric to graph (default is F2 score)
+            """
+            scores = {}
+            scores['entity'] = list(self.results.entity_recall_dict.keys())
+            scores['recall'] = list(self.results.entity_recall_dict.values())
+            scores['precision'] = list(self.results.entity_precision_dict.values())
+            scores['count'] = list(self.results.n_dict.values())
+            scores['f2_score'] = [Evaluator.f_beta(precision=precision, recall=recall, beta=2.5)
```

Comment:
Consider having `beta` as a parameter to this class
```suggestion
            scores[f"f{self.beta}_score"] = [Evaluator.f_beta(precision=precision, recall=recall, beta=self.beta)
```

Reply:
Done